# Robot name goes here
tbot1:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  front_left_bracket_control: # change the name
    type: effort_controllers/JointPositionController # Type of controller. 
    joint: front_left_bracket_joint 
    pid : {p: 1000.0 , i: 0.0 , d: 0.0} 

  # Controller 2
  front_right_bracket_control: # change the name
    type: effort_controllers/JointPositionController # Type of controller. 
    joint: front_right_bracket_joint 
    pid : {p: 1000.0 , i: 0.0 , d: 0.0}
  # Controller 3
  rear_left_wheel_control: # change the name
    type: velocity_controllers/JointVelocityController # Type of controller. 
    joint: rear_left_wheel_joint 
    pid : {p: 1000.0 , i: 0.0 , d: 0.0}
  # Controller 4
  rear_right_wheel_control: # change the name
    type: velocity_controllers/JointVelocityController # Type of controller. 
    joint: rear_right_wheel_joint 
    pid : {p: 1000.0 , i: 0.0 , d: 0.0}
    
  robot_arm_shoulder_pan_control:
    type: effort_controllers/JointPositionController 
    joint: robot_arm_shoulder_pan_joint  
    pid : {p: 1000.0, i: .001, d: 60.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  robot_arm_shoulder_lift_control:
    type: effort_controllers/JointPositionController
    joint: robot_arm_shoulder_lift_joint  
    pid : {p: 3000.0, i: 100.0, d: 175.0, i_clamp_min: -400.0, i_clamp_max: 400.0}
 
  robot_arm_elbow_control:
    type: effort_controllers/JointPositionController
    joint: robot_arm_elbow_joint  
    pid : {p: 2500.0, i: 100.0, d: 100.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  robot_arm_wrist_control_1:
    type: effort_controllers/JointPositionController
    joint: robot_arm_wrist_1_joint  
    pid : {p: 900.0, i: 10.0, d: 40.0, i_clamp_min: -400.0, i_clamp_max: 400.0}

  robot_arm_wrist_control_2:
    type: effort_controllers/JointPositionController 
    joint: robot_arm_wrist_2_joint  
    pid : {p: 900.0, i: 0.1, d: 10.0, i_clamp_min: -100.0, i_clamp_max: 100.0}
    
  robot_arm_wrist_control_3:
    type: effort_controllers/JointPositionController 
    joint: robot_arm_wrist_3_joint  
    pid : {p: 100.0, i: 0.1, d: 10.0, i_clamp_min: -100.0, i_clamp_max: 100.0}
  





























